### PR TITLE
Unitest sort book

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Unitest sort book
## Reference Requirements
- Task ID: #82

## Reference Isuees
#{issue_id}

## Migrations
YES 

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: tests/Browser/Pages/Backend/SortBookTest.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                    ->clickLink("2");
+            $arrAsc = Book::with('category')->orderBy($order, 'desc')->pluck($name)->toArray();
+            $arraySortDesc = array_chunk($arrAsc, 15)[1];
+            for ($i = 1; $i <= 2; $i++) {
+                $selector = "#table-index tbody tr:nth-child($i) td:nth-child($columIndex)";
+                $this->assertEquals($browser->text($selector), $arraySortDesc[$i - 1]);
+            }
+        });
+    }
+
```
Comment (at-haphan) [inline tests/Browser/Pages/Backend/SortBookTest.php]: xóa 1 dòng dư

**Comment by at-haphan** [comment]
LGTM

File: tests/Browser/Pages/Backend/SortBookTest.php [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+     */
+    public function testSortListBooks($name, $order, $columIndex)
+    {
+        $this->browse(function (Browser $browser) use ($name, $order, $columIndex) {
+            $browser->loginAs($this->user)
+                ->visit('admin/books')
+                ->resize(1200, 1600)
+                ->click("#book-sort-$name a");
+            // Test list Asc
+            $arrAsc =Book::with('category')->orderBy($order, 'asc')->pluck($name)->toArray();
```
Comment (at-vietpham) [inline tests/Browser/Pages/Backend/SortBookTest.php]: Thêm dấu cách sau dấu `=`

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by at-vietpham** [comment]
LGTM 🕋

**Pull request merged**